srcpkgs/libcamera/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'libcamera'
+pkgname=libcamera
+version=0.0.3
+revision=1
+build_style=meson
+hostmakedepends="pkg-config python3-yaml python3-ply python3-Jinja2"
+makedepends="openssl-devel libevent-devel gst-plugins-base1-devel gstreamer1-devel libyaml-devel"
+short_desc="Complex camera support library for Linux, Android, and ChromeOS"
+maintainer="Damiano Ognissanti <ognissanti@hotmail.se>"
+license="LGPL-2.1-or-later, GPL-2.0-or-later"
+homepage="https://github.com/kbingham/libcamera"
+distfiles="https://github.com/kbingham/libcamera/archive/v${version}.tar.gz"
```

Comment:
That's a personal fork, we should use the upstream available at https://git.libcamera.org/libcamera/libcamera.git/

Reply:
custom do_check is not acceptable

Reply:
I think that switching back to the github repository might be the best option. Unfortunately they don't provide tarballs on their website and in cgit. The issue with a custom do_fetch is that it's unnecessarily computationally expensive, so it takes a longer time, doesn't get cached, not checking the checkusm, depending on an external tool and not using $distfiles, so the tarball won't get uploaded to sources.voidlinux.org.

Reply:
That's a very good point. Can we nudge upstream to provide tarballs? I'd still want to avoid relying on some random persons private mirror.

Reply:
Alpine seems to be using another unofficial github mirror @ libcamera-org/libcamera: https://git.alpinelinux.org/aports/tree/community/libcamera/APKBUILD